website/source/docs/providers/terraform/r/remote_state.html.md

```diff
@@ -33,6 +33,10 @@ The following arguments are supported:
 * `backend` - (Required) The remote backend to use.
 * `config` - (Optional) The configuration of the remote backend.
 
+When `config` is "atlas" the following sub-arguments apply:
+
+* `name` - (Required) The name of the target Atlas user/organisation and environment, eg: hashicorp/vpc-prod
+
```

Comment:
This is thoroughly described per backend (including Atlas) in a new section of docs: https://www.terraform.io/docs/state/remote/atlas.html

Maybe we can add a link to that page instead? :wink: